tools/BLASTp_Bootcamp.md

```diff
@@ -0,0 +1,115 @@
+# BlastP Bootcamp
+written by: Jade Juba
+
+[20 minutes] **BLASTp, the Basic Local Alignment Search Tool for proteins**, takes a "query" protein sequence, breaks it into pieces, and compares each piece to a database of known proteins. Once it finds a match, it extends the alignment until the entire query sequence is accounted for, then it returns all significant matches it finds. BLASTp can be used to compare sequences, characterize sequences of interest, find proteins with functional and/or structural similarities, and more! After generating a putative open reading frame for your obelisk and decoding the codons into amino acid residues, you can use BLASTp to investigate the oblin(s) it codes for.
+
+**Tutorial Objective**: We will use `BLASTp` to investigate our Oblin and look for similar amino acid sequences.
+
+## Input / Prerequisites
+- Access to [BLASTp](https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE=Proteins)
+- Access to [ORFfinder](https://www.ncbi.nlm.nih.gov/orffinder/), to generate putative ORFs and amino acid sequences from your obelisk
+- Web browser
+- A protein sequence
+
+Find example data [here](https://www.uniprot.org/uniprotkb/O89281/entry). This link will send you to the `Uniprot` page for the `RdRp catalytic domain-containing protein` of `Tremovirus A`, a virus that infects chickens and some other fowl. On the left-hand menu, click `Sequence`, and in the light-blue box containing the amino acid sequence, click `Copy sequence`. You'll be using it in your tutorial-guided analysis.
+
+## Output
+
+The identification of `amino acid sequences` which have some similarity to our oblin sequence.
+
+### 1. Navigate to the [BLASTp portal](https://blast.ncbi.nlm.nih.gov/Blast.cgi?PAGE=Proteins)
+
+### 2. Create and submit a protein search query
+
+**2.1: Entering our amino acid sequence**
+
+Start by pasting your **amino acid sequence** in the box under the `Enter Query Sequence` titlecard. Make sure to title your sequence with a single line above the sequence, starting with a greater-than sign `>`. This is known as FASTA format. I named my sequence `obelisk223_putative_oblin`. If you have a single sequence, the title you enter will also appear in the `Job Title` box.
+
+Instead of the amino acid sequence, you can also paste an accession number connected to your sequence of interest in a database, or you can upload your sequence to the system as a file (using the `Upload File` button underneath the text box).
+
+![](img/BlastP_Bootcamp/enter_sequence.jpg)
```

Comment:
It's best practice to include a "Figure Title" incase the image does not load, so that backup text is displayed.